### PR TITLE
test folders flattering

### LINKED ISSUE #137: Ungroup the test subfolders
In the tests folder https://github.com/StarcounterApps/KitchenSink/tree/1b8b41aea334bc491f6c1afbf62ce70b38c63421/test/KitchenSink.Tests/Test, there are subfolders that correspond to the NavPage menu groups that are no longer in place.

I suggest to flatten the tests folders, just alike the viewmodels folder has a flat file structure too.

### PR DESCRIPTION
**Pull request by bouguila**
Fix #137
changed test folders structure

### CODE REVIEW

**Comment by warpech** [comment]
@patrykszwer could you pls review?

**Comment by warpech** [comment]
Ping @PatrykSzwer

**Comment by warpech** [comment]
Ping @bouguila

**Comment by warpech** [comment]
The tests are now passing.

@bouguila please merge branch `develop` to this feature branch. You have merged `master`.

@PatrykSzwer is this ok now?

**Comment by alshakero** [comment]
If you merge into `develop` now there is a possibility of making a mess, because this branch now is polluted with traces from `master`.

Safer to rollback the last two commits. And then manually apply [this diff](https://github.com/StarcounterApps/KitchenSink/pull/142/commits/151b668a773316032519a5b42df6fcf7364177b0) to `test/KitchenSink.Tests/Test/ButtonPageTest.cs`. Push a commit to `test-folders-ungrouping-#137`. See if tests pass. Then merge into **`develop`**.

**Comment by warpech** [comment]
Are you sure? `master` is just few commits behind `develop`, the branches are not diverged. I think it is fine to merge as is, if @PatrykSzwer has no more comments

**Comment by bouguila** [comment]
ping @PatrykSzwer 
what do you thinK?

**Comment by alshakero** [comment]
@warpech I am not sure at all. I just noticed an unintended merge from `master`. And thought I should warn him not to commit the mistake I did on Level1 earlier. Just helping out a teammate 🥉 

If `master` is not different there is no need obviously.

File: test/KitchenSink.Tests/Ui/MainPage.cs [hunk 1]
```diff
@@ -81,12 +75,6 @@ public AutoCompletePage GoToAutoCompletePage()
             return new AutoCompletePage(Driver);
         }
 
-        public ClientLocalStatePage GoToClientLocalStatePage()
```
Comment (PatrykSzwer) [inline test/KitchenSink.Tests/Ui/MainPage.cs]: Probably will be needed for missing files.

**Comment by bouguila** [comment]
@PatrykSzwer you were right about the missing files. thanks!
Can we merge this branch now since the tests are passing?

**Pull request merged**